main/models.py

```diff
@@ -159,6 +163,6 @@ def process_faculties(self, faculties):
                     continue
 
                 faculty_obj.users.add(self)
-            except:
-                # Just ignore any errors...
+            except Exception as e:
+                logger.error(f"Error processing faculty for user: {e}", exc_info=True)
```

Comment:
I kind of half-remember it being best practice to not format log strings actively, but to use the formatting provided by `logger.error()` itself. Like that the logging handlers and formatters can work their magic.

But I can't find a reference to that anywhere so it might not even be the case anymore. Not blocking issue either way.

Reply:
Just to add my 5 cents here (I don't know why I'm getting email notifications from this PR): Pylint was mad at me last week for using f-strings instead of the old format: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation. Maybe you are thinking about that?

Reply:
Yeah y'all are right that this isn't the ideal/correct way to do this. (One point not even mentioned is me formatting an exception into a string, which is definitely the worse offense here actually). 

The method I used (f-strings) was me being lazy. The reason (doing it at all) was because I wasn't sure if Sentry would correctly capture the whole exception context. (That's what `exc_info` does btw).

I just did a [local test](https://centre-for-digital-humanities.sentry.io/issues/5248791949/?query=is%3Aunresolved&referrer=issue-stream&stream_index=0), and it does! As this makes the string-format unnecessary, I removed it